provider/kubernetes/kubernetes.go

```diff
@@ -247,8 +247,14 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					templateObjects.Backends[r.Host+pa.Path].LoadBalancer.Method = "drr"
 				}
 
-				if service.Annotations[types.LabelBackendLoadbalancerSticky] == "true" {
-					templateObjects.Backends[r.Host+pa.Path].LoadBalancer.Sticky = true
+				if len(service.Annotations[types.LabelBackendLoadbalancerSticky]) != 0 {
```

Comment:
`len(service.Annotations[types.LabelBackendLoadbalancerSticky]) > 0` would be even better :nerd_face:

Reply:
Do you have any reference on this topics?